src/index.html

```diff
@@ -1,12 +1,340 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+        src="./images/logo.png"
+        alt="Moyo logo"
+      >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__items">
+            <a
+              class="nav__cell is-active"
+              href="#apple"
+              data-qa="card"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__items">
+            <a class="nav__cell" href="#samsung">Samsung</a>
```

Comment:
Fix classes everywhere
```suggestion
          <li class="nav__item">
            <a class="nav__linkl" href="#samsung">Samsung</a>
```